scripts/local/start_network.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+set -eu
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+
+source $SCRIPT_DIR/vars.sh
+
+mkdir -p $SCRIPT_DIR/logs
+
+cache="${1:-false}"
+
+STRIDE_STATE=$SCRIPT_DIR/state/stride
+STRIDE_LOGS=$SCRIPT_DIR/logs/stride.log
+GAIA_STATE=$SCRIPT_DIR/state/gaia
+GAIA_LOGS=$SCRIPT_DIR/logs/gaia.log
+HERMES_LOGS=$SCRIPT_DIR/logs/hermes.log
+ICQ_LOGS=$SCRIPT_DIR/logs/icq.log
+
+if [ "$cache" == "true" ]; then
+    echo "Restoring from cache..."
+    rm -rf $SCRIPT_DIR/state 
+    mv $SCRIPT_DIR/.state.backup $SCRIPT_DIR/state
+fi
+
+printf '%s' "Starting Stride and Gaia...   "
+nohup $STRIDE_CMD start | sed -r -u "s/\x1B\[([0-9]{1,3}(;[0-9]{1,2})?)?[mGK]//g" > $STRIDE_LOGS 2>&1 &
```

Comment:
are these the lines that wait until blocks are committed? if so, very cool! 

what does the sed command do on this line? and does the `tail` line below wait until the grep command returns something? i would have thought this requires a while loop, v neat if not